app/models/package_manager/npm.rb

```diff
@@ -145,16 +145,22 @@ def self.one_version(raw_project, version_string)
         .find { |v| v[:number] == version_string }
     end
 
-    def self.dependencies(_name, version, mapped_project)
+    def self.dependencies(name, version, mapped_project)
```

Comment:
separate bugfix: since we called this `_name` before, the logger below on line 151 was actually logging the **platform** name instead of the package name.